doc/refman/Universes.tex

```diff
@@ -139,7 +139,9 @@
 Inductive types, coinductive types, variants and records can be
 declared cumulative using the \texttt{Cumulative}. Alternatively,
 there is an option \texttt{Set Inductive Cumulativity} which when set,
-makes all subsequent inductive definitions cumulative. Consider the examples below.
+makes all subsequent inductive definitions cumulative.  When set
+inductive types and the like can be enforced to be
```

Comment:
Add a comma after "set", otherwise I parse it as "set inductive types" and get confused